packages/styles/src/components/alert.scss

```diff
@@ -45,10 +45,7 @@
     flex: 0 0 auto;
     display: flex;
     align-items: center;
-
-    > .btn {
-      margin-inline-start: notification.$notification-buttons-margin-start;
-    }
+    gap: notification.$notification-buttons-gap;
```

Comment:
Does it require to be in the changelog for styles?

Reply:
I added one.